src/core/backbone_router/leader.cpp

```diff
@@ -137,7 +149,44 @@ const char *Leader::StateToString(State aState)
     return logString;
 }
 
+const char *Leader::StateToString(DomainPrefixState aState)
+{
+    const char *logString = "Unknown";
+
+    switch (aState)
+    {
+    case kDomainPrefixNone:
+        logString = "Domain Prefix: None";
```

Comment:
In `LogDomainPrefix` above, we already include `Domain Prefix: `. Should we remove redundancy?

Reply:
Good catch!